{{cookiecutter.project_shortname}}/code/.gitkeep

```diff
@@ -0,0 +1 @@
+# This file exists only to add templates/ folder to git repository.
```

Comment:
Major: in our IRL discussion, we also talked about having sub-folders under `code/` in order to

- have some namespacing;
- be able to e.g. symlink other packages under the `code/` directory;
- in a Docker image (for development) one could install everything like `for p in code/*; do pip install -e $d; done`.

That would mean that by default cookiecutter would generate `{{cookiecutter.project_shortname}}/code/{{cookiecutter.project_package}}/.gitkeep`, to leave space for other folders/packages.